src/main/java/org/valdroz/vscript/BaseNode.java

```diff
@@ -510,6 +514,55 @@ public Variant execute(VariantContainer variantContainer) {
                 }
                 break;
 
+            case NT_MF_FORMAT_TS:
+                if (params == null || params.size() < 2 || params.size() > 3) {
+                    throw new EvaluationException(String.format("Function %1$s` takes 2 or 3 parameters. " +
+                                    "E.g. %1$s(\"2023-02-28T05:16:55.835697363Z\", \"mm/dd/yy\") will result in \"02/28/23\"",
+                            EquationParser.functionNameFromCode(operation)));
+                }
+
+                String tsToBeFormatted = getParameterOrNullNode(0).execute(variantContainer).asString();
+
+                if (StringUtils.isBlank(tsToBeFormatted)) {
+                    return Variant.nullVariant();
+                }
+
+                DateTime ts;
+                try {
+                    if (StringUtils.isNumeric(tsToBeFormatted)) {
+                        ts = new DateTime(Long.valueOf(tsToBeFormatted));
+                    } else {
+                        ts = new DateTime(tsToBeFormatted);
+                    }
+                } catch(IllegalArgumentException iae) {
+                    throw new EvaluationException("Invalid timestamp.");
```

Comment:
Message should also include details about the function name and what parameter the error was linked to.